### PR TITLE
issues-8: Add find-a-doctor style

### PR DESCRIPTION
**Pull request by QuentinVecchio**
This PR introduces a new block for the find-a-doctor page.
For now only the form is rendered, there is no integration yet with the API.

Part of #8 

Test URLs:
- Before: https://main--durysta--hlxsites.hlx.page/find-a-doctor
- After: https://8-find-doctor--durysta--hlxsites.hlx.page/find-a-doctor

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -127,6 +127,10 @@
   --list-marker-color: var(--clr-orange-brown);
   --columns-button-color: var(--clr-yellow);
   --conditions-background-color: #f4f4f4;
+  --find-a-doctor-submit-color: #000;
```
Comment (kmurugulla) [inline styles/styles.css:130]: looks like we need -clr-black variable with value #000 as i see it appearing twice

File: blocks/find-a-doctor/find-a-doctor.css [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    text-transform: uppercase;
+    display: block;
+}
+
+.find-a-doctor .columns-col:nth-of-type(2) {
+    display: none;
+}
+
+.find-a-doctor .preloader {
+    border: 5px solid #00806e;
```
Comment (kmurugulla) [inline blocks/find-a-doctor/find-a-doctor.css:142]: good opportunity to use the variable --clr-green :)

**Review by kmurugulla — CHANGES_REQUESTED** [review]
Requesting couple of minute changes to leverage css variables

File: blocks/find-a-doctor/find-a-doctor.css (hunk 2 above)
Comment (QuentinVecchio) [reply blocks/find-a-doctor/find-a-doctor.css]: Fixed

**Review by kmurugulla — APPROVED** [review]

**Pull request merged**